docs/_includes/css/store.css

```diff
@@ -0,0 +1,243 @@
+.hero {
```

Comment:
make this an `.scss` file, so that it is minified and the browser prefixes are put automatically. And it would be easier to read and write.